src.ts/utils/format.ts

```diff
@@ -0,0 +1,37 @@
+const BN = require ('bn.js');
+
+// Exponent for calculating how many units of account balance are in one near.
+const exp = 24;
+// actual number of units of account balance in one near.
+const unitsInOneNear = new BN('10', 10).pow(new BN(exp, 10));
+
+/**
+ * Convert account balance to near.
+ * @param balance
+ */
+export function convertAccountBalanceAmountToNear(balance: string): string {
```

Comment:
`balance` argument should better support `BN` as well

Reply:
discussed offline.